firebase/BUILD.bazel

```diff
@@ -0,0 +1,26 @@
+load("@rules_jvm_external//:defs.bzl", "artifact")
```

Comment:
We can have some top-level documentation to understand what this directory is all about.

Reply:
Done!

Reply:
Nit changes:
```
'''
Builds and exports Firebase dependencies.
To reference these dependencies, add '//firebase:crashlytics_lib' and '//firebase:crashlytics_deps'
to your build rule's dependency list.
'''
```

Reply:
Done!